src/main/resources/application.properties

```diff
@@ -2,6 +2,6 @@ spring.datasource.url=jdbc:mysql://localhost:3306/customer_manager
 spring.datasource.username=root
 spring.datasource.password=S@xena1216
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=create
```

Comment:
I hope this is due to the fact that if you do any changes in db then it is dropped and created again?

Reply:
Reverting this one for higher env.